deps.windows/70-vpl.ps1

```diff
@@ -0,0 +1,87 @@
+param(
+    [string] $Name = 'vpl',
+    [string] $Version = 'v2023.3.0',
+    [string] $Uri = 'https://github.com/oneapi-src/oneVPL.git',
+    [string] $Hash = 'e12ace9761bb52786409e830f619916b86e87fc5'
+)
+
+function Setup {
+    Setup-Dependency -Uri $Uri -Hash $Hash -DestinationPath $Path
+}
+
+function Clean {
+    Set-Location $Path
+
+    if ( Test-Path "build_${Target}" ) {
+        Log-Information "Clean build directory (${Target})"
+        Remove-Item -Path "build_${Target}" -Recurse -Force
+    }
+}
+
+function Configure {
+    Log-Information "Configure (${Target})"
+    Set-Location $Path
+
+    $Options = @(
+        $CmakeOptions
+        '-DBUILD_DISPATCHER_ONLY:BOOL=ON'
+		'-DBUILD_EXAMPLES:BOOL=OFF'
+		'-DBUILD_DISPATCHER_ONEVPL_EXPERIMENTAL_ONLY:BOOL=OFF'
+		'-DBUILD_SHARED_LIBS:BOOL=OFF'
```

Comment:
@PatTheMav should this use the $OnOff syntax to be controllable rather than hard-coded?

Reply:
Should use the existing pattern instead of hardcoding. We might revise this in the future as we have a growing number of dependencies with different usage requirements (some can _only_ be used as shared libraries, some can _only_ be used as static ones, some are _preferred_ to be used as shared libraries, and so on), but that's the requirement for now.

Reply:
I've addressed this via force push. Resolving this thread.